scavenger-agent-java/src/integrationTest/java/sample/app/SampleApp.java

```diff
@@ -0,0 +1,48 @@
+package sample.app;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.stereotype.Controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.java.Log;
+
+/**
+ * @author olle.hallin@crisp.se
+ */
+@SuppressWarnings("ALL")
+@Log
+@SpringBootApplication
+@EnableAspectJAutoProxy
+@RequiredArgsConstructor
+@Controller
+public class SampleApp {
+    private final int dummy = 17;
+    private final SampleService1 sampleService1;
+
+    public static void main(String[] args) throws InterruptedException {
+        log.info(
+            String.format(
+                "%s starts on Java %s",
+                SampleApp.class.getSimpleName(), System.getProperty("java.version")));
```

Comment:
Can we make it just like
```
log.info("{} starts on Java {}", SampleApp.class.getSimpleName(),System.getProperty("java.version")
)
```

I don't know which actual logger is used under the Log.

Reply:
It uses the default Java logger `java.util.logging.Logger`, which only takes one `String` value as a parameter. I've changed this to use the plain string concatenation `+` without using `String.format()` in [fe07c0f](https://github.com/naver/scavenger/pull/71/commits/fe07c0fa0198503b576deb9b6af3a69701702650).